examples/README.md

```diff
@@ -44,6 +44,8 @@ We start with AWS KMS examples, then show how to use other wrapping keys.
         * [with keyrings](./src/keyring/aws_kms/discovery_decrypt_in_region_only.py)
     * How to decrypt with a preferred region but failover to others
         * [with keyrings](./src/keyring/aws_kms/discovery_decrypt_with_preferred_regions.py)
+    * How to replicate the behavior of an AWS KMS master key provider
```

Comment:
Can we use "reproduce" instead of "replicate?" The KMS is using "replicate" and "replica" for multi-region keys. Although most people won't notice, I can envisions someone with limited English language skills googling this and becoming hopelessly confused.